201-cdn-oms-solution/azuredeploy.json

```diff
@@ -0,0 +1,582 @@
+{
+  "$schema": "http://schemas.microsoft.org/azure/deploymentTemplate?api-version=2015-01-01#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "A valid Log Analytics workspace name"
+      }
+    },
+    "accountName": {
```

Comment:
I don't see any references to automation resources in this template, so this parameter can be removed.